README.md

```diff
@@ -1,6 +1,11 @@
 ## Amazon Neptune Sparql Java Sigv4
 
-A SPARQL client for Amazon Neptune that includes AWS Signature Version 4 signing. Implemented as an RDF4J repository. 
+A SPARQL client for Amazon Neptune that includes AWS Signature Version 4 signing. Implemented as an RDF4J repository and Jena HTTP Client.
+ 
+For example usage refer to:
+ 
+-	[NeptuneJenaSigV4Example.java](https://github.com/aws/amazon-neptune-sparql-java-sigv4/blob/master/src/main/java/com/amazonaws/neptune/client/jena/NeptuneJenaSigV4Example.java) 
+-	[NeptuneRdf4JSigV4Example.java](https://github.com/aws/amazon-neptune-sparql-java-sigv4/blob/master/src/main/java/com/amazonaws/neptune/client/rdf4j/NeptuneRdf4JSigV4Example.java) 
 
```

Comment:
Please add a reference and link back to the Neptune docs: https://docs.aws.amazon.com/neptune/latest/userguide/iam-auth-connecting-sparql-java.html.